varya/assets/sass/abstracts/_config.scss

```diff
@@ -44,7 +44,7 @@ $typescale-ratio: 1.2; // Run ratio math on 1em == $typescale-base * $typescale-
 	--global--color-background-light: #FAFBF6;
 	--global--color-background-dark: #DDDDDD;
 	--global--color-border: #757575;
-	--global--color-text-selection: #FAFBF6;
+	--global--color-text-selection: rgba( 163, 98, 101, 0.1 ); // --global--color-secondary in RGB, at 10% opacity
```

Comment:
I would've preferred to automate this in some way, e.g.: 

```
--global--color-text-selection: rgba( --global--color-secondary, 0.1 );
```

... however this only works if you define the original color variable via an RGB value, not Hex. We could do that here, but I worry that a child them might define the color in Hex, think it's working fine, but then be confused when there's no selection color at all. 😕 So I think this is the best route for now.

Reply:
That could be an interesting idea to pose to the global styles crew. Since the global styles mechanism introduces a processing step (parsing the JSON file), certain color values could be computed / generated.

Reply:
This is super interesting, but I think your right about possible confusion. There might be some PostCSS thing we could do to convert Hex to RGBa for everything but that’d probably be overkill. 

Also agreed that this would be a really clever thing for global styles to handle 👍.